cypress/utils/apps/query_enhancements/commands.js

```diff
@@ -3,7 +3,80 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-Cypress.Commands.add('setQueryEditor', (value, opts = {}, submit = true) => {
```

Comment:
This `opts` is needed for the ignoreSpecialSequence or whatever it was called

Reply:
Will add